tests/sentry/api/test_event_search.py

```diff
@@ -1149,6 +1149,14 @@ def test_transaction_status_invalid(self):
         assert "Invalid value" in six.text_type(err)
         assert "cancelled," in six.text_type(err)
 
+    def test_general_user_field(self):
+        conditions = get_filter("user:123").conditions
```

Comment:
I checked snuba and all the user promoted columns are strings so we won't hit any type incompatibilities in events, but the ip_address column in transactions might fail with strings that can't be converted into an iPhone address.

Reply:
Hmm @markstory I might be misunderstanding you, but is the concern that `user.ip = not_an_ip` gonna result in an issue? 

If so I think `test_organization_events_v2.py:test_user_search` checks that since it'll search for a transaction with `user.ip:foo@example.com`

Reply:
I missed that :+1:

Reply:
🎉 good to check anyways :+1: